openfisca_us/variables/finance/benefit/outputs.py

```diff
@@ -0,0 +1,67 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class snap(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = u"Supplemental Nutrition Assistance Program, benefit amount"
+    definition_period = YEAR
+
+    def formula(spm_unit, period, parameters):
+
+        # Params:
+        # Income Threshold
+        # Max Benefit
+        # Phase-out Rate
+        # for later, Employment Rules
+
+        # Get vars:
+        # net income
+        # gross income
+        # number of individuals
+
+        # for later, Employment Status
+
+        return standard
+
+
+class expected_food_contribution(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = u"Supplemental Nutrition Assistance Program, spm unit's expected food contribution"
+    definition_period = YEAR
+
+    def formula(spm_unit, period, parameters):
+
+        # Params:
+        # Income Threshold
+        # Max Benefit
+        # Phase-out Rate
+        # for later, Employment Rules
+
+        # Get vars:
+        # net income
+        # gross income
+        # number of individuals
+
+        # for later, Employment Status
+
+        return standard
+
+
+class maximum_snap_benefit(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = u"Supplemental Nutrition Assistance Program, spm unit's max possible unit based on family size"
+    definition_period = YEAR
+
+    def formula(spm_unit, period, parameters):
+
+        max_by_household_size = parameters(
```

Comment:
This applies to lots of other instances (including on the tax side), but bear in mind that because this formula is calculatable for any year after 0 AD (`formula(` is effectively `formula_0000(`), the model at the moment will break if trying to calculate anything involving this variable for years before 2021, because it won't find the parameter for that year. I'd suggest changing `formula` to `formula_2021` until we decide to add older values. This'll make SNAP zero before 2021.